packages/kbn-apm-synthtrace/src/lib/logs/logs_synthtrace_es_client.ts

```diff
@@ -60,6 +64,38 @@ export class LogsSynthtraceEsClient extends SynthtraceEsClient<LogDocument> {
       this.logger.error(`Index creation failed: ${index} - ${err.message}`);
     }
   }
+
+  async updateIndexTemplate(
+    indexName: string,
+    modify: (
+      template: ValuesType<
+        estypes.IndicesGetIndexTemplateResponse['index_templates']
+      >['index_template']
+    ) => estypes.IndicesPutIndexTemplateRequest
+  ) {
+    const response = await this.client.indices.getIndexTemplate({
+      name: indexName,
+    });
+
+    await Promise.all(
+      response.index_templates.map((template) => {
+        return this.client.indices.putIndexTemplate({
+          ...modify(template.index_template),
+          name: template.name,
+        });
+      })
+    );
+
+    this.logger.info(`Updated ${indexName} index template`);
+  }
+
+  async createCustomPipeline(processors: IngestProcessorContainer[]) {
+    this.client.ingest.putPipeline({
+      id: LogsCustom,
```

Comment:
The idea is to add the custom changes to that pipeline instead of trying to update the default one. I think it's fine, if we need to update other pipelines we can refactor and expect the name also as param